schema/meta_schema_types.yaml

```diff
@@ -1264,3 +1266,48 @@
     - enumValue: WARN4
       description: WARN4, severity number 16.
   isSdkExtensionPlugin: false
+- type: SpanKind
+  enumValues:
+    - enumValue: CLIENT
+      description: CLIENT, a client span.
+    - enumValue: CONSUMER
+      description: CONSUMER, a consumer span.
+    - enumValue: INTERNAL
+      description: INTERNAL, an internal span.
+    - enumValue: PRODUCER
+      description: PRODUCER, a producer span.
+    - enumValue: SERVER
+      description: SERVER, a server span.
+  isSdkExtensionPlugin: false
+- type: ExperimentalComposableRuleBasedSampler
+  properties:
+    - property: rules
+      description: The rules for the sampler, matched in order. Each rule can have multiple match conditions - the sampler will be applied if all match. If no conditions are specified, the rule matches all spans that reach it.
```

Comment:
Probably worth describing that if no rules are matched, the sampler uses a non-sampling intent: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#composablerulebased